CHANGELOG.md

```
@@ -1,3 +1,11 @@
+## [9.29.2] - 2026-04-23
```

markdownlint-cli2 (0.22.1)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)